custom_components/rct_power/lib/entities.py

```diff
@@ -232,6 +232,12 @@ def get_matching_names(expression: str):
         name="Heat Sink Temperature",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    RctPowerSensorEntityDescription(
+        get_device_info=get_inverter_device_info,
+        key="db.temp2",
+        name="Heat Sink (battery actuator) Temperature",
```

Comment:
How about we get rid of the parentheses and call this

```suggestion
        name="Battery Actuator Temperature",
```

?

Reply:
I know the name is long and not nice. But it's the name as it is presented in the official app and also the python library. https://rctclient.readthedocs.io/en/latest/inverter_app_mapping.html#inverter

Personally, I don't mind which name we will use, maybe I like the shorter one better. However, I'm not sure if it will be confusing. WDYT?

Reply:
> If I remember correctly there is one big heat sink in the inverter and this seems to be the temperature of the part that is closer to the battery actuator.

I can follow that argument, thanks for explaining. I wasn't aware of the physical arrangement in the inverter. So I'd be fine with leaving it the way you proposed here.